packages/stellar_solana_bridge/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/favicon.png" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>TFT Solana</title>
```

Comment:
please update this title
```suggestion
    <title>Stellar-Solana Bridge</title>
```

Reply:
they requested this change in the demo